pandas/core/generic.py

```diff
@@ -4522,6 +4530,102 @@ def copy(self, deep=True):
         Returns
         -------
         copy : type of caller
+
+        Examples
+        --------
+        >>> s = pd.Series([1,2], dtype="int32", index=["a", "b"])
+        >>> s
+        a    1
+        b    2
+        dtype: int32
+        >>> s_copy = s.copy()
+        >>> s_copy
+        a    1
+        b    2
+        dtype: int32
+
+        Shallow copy versus default (deep) copy:
+
+        In a shallow copy the index and data is a reference to the original
+        objects'.
+
+        >>> s = pd.Series([1,2], dtype="int32", index=["a", "b"])
+        >>> deep_copy = s.copy()
+        >>> shallow_copy = s.copy(deep=False)
+        >>> id(s) == id(shallow_copy)
```

Comment:
These examples are certainly comprehensive but perhaps too verbose. Can you refactor to make the index / values comparisons fit on one line? Similar comment with the value assignment - any consolidation will make it more readable